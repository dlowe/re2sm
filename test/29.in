((s|t|o|p)?[doing]{1,2})[that]
