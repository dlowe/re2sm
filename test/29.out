S3
Rda	$@<MATCH>
Rdh	$@<MATCH>
Rdt	$@<MATCH>
Rga	$@<MATCH>
Rgh	$@<MATCH>
Rgt	$@<MATCH>
Ria	$@<MATCH>
Rih	$@<MATCH>
Rit	$@<MATCH>
Rna	$@<MATCH>
Rnh	$@<MATCH>
Rnt	$@<MATCH>
Roa	$@<MATCH>
Roh	$@<MATCH>
Rot	$@<MATCH>
Rdda	$@<MATCH>
Rddh	$@<MATCH>
Rddt	$@<MATCH>
Rdga	$@<MATCH>
Rdgh	$@<MATCH>
Rdgt	$@<MATCH>
Rdia	$@<MATCH>
Rdih	$@<MATCH>
Rdit	$@<MATCH>
Rdna	$@<MATCH>
Rdnh	$@<MATCH>
Rdnt	$@<MATCH>
Rdoa	$@<MATCH>
Rdoh	$@<MATCH>
Rdot	$@<MATCH>
Rgda	$@<MATCH>
Rgdh	$@<MATCH>
Rgdt	$@<MATCH>
Rgga	$@<MATCH>
Rggh	$@<MATCH>
Rggt	$@<MATCH>
Rgia	$@<MATCH>
Rgih	$@<MATCH>
Rgit	$@<MATCH>
Rgna	$@<MATCH>
Rgnh	$@<MATCH>
Rgnt	$@<MATCH>
Rgoa	$@<MATCH>
Rgoh	$@<MATCH>
Rgot	$@<MATCH>
Rida	$@<MATCH>
Ridh	$@<MATCH>
Ridt	$@<MATCH>
Riga	$@<MATCH>
Righ	$@<MATCH>
Rigt	$@<MATCH>
Riia	$@<MATCH>
Riih	$@<MATCH>
Riit	$@<MATCH>
Rina	$@<MATCH>
Rinh	$@<MATCH>
Rint	$@<MATCH>
Rioa	$@<MATCH>
Rioh	$@<MATCH>
Riot	$@<MATCH>
Rnda	$@<MATCH>
Rndh	$@<MATCH>
Rndt	$@<MATCH>
Rnga	$@<MATCH>
Rngh	$@<MATCH>
Rngt	$@<MATCH>
Rnia	$@<MATCH>
Rnih	$@<MATCH>
Rnit	$@<MATCH>
Rnna	$@<MATCH>
Rnnh	$@<MATCH>
Rnnt	$@<MATCH>
Rnoa	$@<MATCH>
Rnoh	$@<MATCH>
Rnot	$@<MATCH>
Roda	$@<MATCH>
Rodh	$@<MATCH>
Rodt	$@<MATCH>
Roga	$@<MATCH>
Rogh	$@<MATCH>
Rogt	$@<MATCH>
Roia	$@<MATCH>
Roih	$@<MATCH>
Roit	$@<MATCH>
Rona	$@<MATCH>
Ronh	$@<MATCH>
Ront	$@<MATCH>
Rooa	$@<MATCH>
Rooh	$@<MATCH>
Root	$@<MATCH>
Rsda	$@<MATCH>
Rsdh	$@<MATCH>
Rsdt	$@<MATCH>
Rsga	$@<MATCH>
Rsgh	$@<MATCH>
Rsgt	$@<MATCH>
Rsia	$@<MATCH>
Rsih	$@<MATCH>
Rsit	$@<MATCH>
Rsna	$@<MATCH>
Rsnh	$@<MATCH>
Rsnt	$@<MATCH>
Rsoa	$@<MATCH>
Rsoh	$@<MATCH>
Rsot	$@<MATCH>
Rsdda	$@<MATCH>
Rsddh	$@<MATCH>
Rsddt	$@<MATCH>
Rsdga	$@<MATCH>
Rsdgh	$@<MATCH>
Rsdgt	$@<MATCH>
Rsdia	$@<MATCH>
Rsdih	$@<MATCH>
Rsdit	$@<MATCH>
Rsdna	$@<MATCH>
Rsdnh	$@<MATCH>
Rsdnt	$@<MATCH>
Rsdoa	$@<MATCH>
Rsdoh	$@<MATCH>
Rsdot	$@<MATCH>
Rsgda	$@<MATCH>
Rsgdh	$@<MATCH>
Rsgdt	$@<MATCH>
Rsgga	$@<MATCH>
Rsggh	$@<MATCH>
Rsggt	$@<MATCH>
Rsgia	$@<MATCH>
Rsgih	$@<MATCH>
Rsgit	$@<MATCH>
Rsgna	$@<MATCH>
Rsgnh	$@<MATCH>
Rsgnt	$@<MATCH>
Rsgoa	$@<MATCH>
Rsgoh	$@<MATCH>
Rsgot	$@<MATCH>
Rsida	$@<MATCH>
Rsidh	$@<MATCH>
Rsidt	$@<MATCH>
Rsiga	$@<MATCH>
Rsigh	$@<MATCH>
Rsigt	$@<MATCH>
Rsiia	$@<MATCH>
Rsiih	$@<MATCH>
Rsiit	$@<MATCH>
Rsina	$@<MATCH>
Rsinh	$@<MATCH>
Rsint	$@<MATCH>
Rsioa	$@<MATCH>
Rsioh	$@<MATCH>
Rsiot	$@<MATCH>
Rsnda	$@<MATCH>
Rsndh	$@<MATCH>
Rsndt	$@<MATCH>
Rsnga	$@<MATCH>
Rsngh	$@<MATCH>
Rsngt	$@<MATCH>
Rsnia	$@<MATCH>
Rsnih	$@<MATCH>
Rsnit	$@<MATCH>
Rsnna	$@<MATCH>
Rsnnh	$@<MATCH>
Rsnnt	$@<MATCH>
Rsnoa	$@<MATCH>
Rsnoh	$@<MATCH>
Rsnot	$@<MATCH>
Rsoda	$@<MATCH>
Rsodh	$@<MATCH>
Rsodt	$@<MATCH>
Rsoga	$@<MATCH>
Rsogh	$@<MATCH>
Rsogt	$@<MATCH>
Rsoia	$@<MATCH>
Rsoih	$@<MATCH>
Rsoit	$@<MATCH>
Rsona	$@<MATCH>
Rsonh	$@<MATCH>
Rsont	$@<MATCH>
Rsooa	$@<MATCH>
Rsooh	$@<MATCH>
Rsoot	$@<MATCH>
Rtda	$@<MATCH>
Rtdh	$@<MATCH>
Rtdt	$@<MATCH>
Rtga	$@<MATCH>
Rtgh	$@<MATCH>
Rtgt	$@<MATCH>
Rtia	$@<MATCH>
Rtih	$@<MATCH>
Rtit	$@<MATCH>
Rtna	$@<MATCH>
Rtnh	$@<MATCH>
Rtnt	$@<MATCH>
Rtoa	$@<MATCH>
Rtoh	$@<MATCH>
Rtot	$@<MATCH>
Rtdda	$@<MATCH>
Rtddh	$@<MATCH>
Rtddt	$@<MATCH>
Rtdga	$@<MATCH>
Rtdgh	$@<MATCH>
Rtdgt	$@<MATCH>
Rtdia	$@<MATCH>
Rtdih	$@<MATCH>
Rtdit	$@<MATCH>
Rtdna	$@<MATCH>
Rtdnh	$@<MATCH>
Rtdnt	$@<MATCH>
Rtdoa	$@<MATCH>
Rtdoh	$@<MATCH>
Rtdot	$@<MATCH>
Rtgda	$@<MATCH>
Rtgdh	$@<MATCH>
Rtgdt	$@<MATCH>
Rtgga	$@<MATCH>
Rtggh	$@<MATCH>
Rtggt	$@<MATCH>
Rtgia	$@<MATCH>
Rtgih	$@<MATCH>
Rtgit	$@<MATCH>
Rtgna	$@<MATCH>
Rtgnh	$@<MATCH>
Rtgnt	$@<MATCH>
Rtgoa	$@<MATCH>
Rtgoh	$@<MATCH>
Rtgot	$@<MATCH>
Rtida	$@<MATCH>
Rtidh	$@<MATCH>
Rtidt	$@<MATCH>
Rtiga	$@<MATCH>
Rtigh	$@<MATCH>
Rtigt	$@<MATCH>
Rtiia	$@<MATCH>
Rtiih	$@<MATCH>
Rtiit	$@<MATCH>
Rtina	$@<MATCH>
Rtinh	$@<MATCH>
Rtint	$@<MATCH>
Rtioa	$@<MATCH>
Rtioh	$@<MATCH>
Rtiot	$@<MATCH>
Rtnda	$@<MATCH>
Rtndh	$@<MATCH>
Rtndt	$@<MATCH>
Rtnga	$@<MATCH>
Rtngh	$@<MATCH>
Rtngt	$@<MATCH>
Rtnia	$@<MATCH>
Rtnih	$@<MATCH>
Rtnit	$@<MATCH>
Rtnna	$@<MATCH>
Rtnnh	$@<MATCH>
Rtnnt	$@<MATCH>
Rtnoa	$@<MATCH>
Rtnoh	$@<MATCH>
Rtnot	$@<MATCH>
Rtoda	$@<MATCH>
Rtodh	$@<MATCH>
Rtodt	$@<MATCH>
Rtoga	$@<MATCH>
Rtogh	$@<MATCH>
Rtogt	$@<MATCH>
Rtoia	$@<MATCH>
Rtoih	$@<MATCH>
Rtoit	$@<MATCH>
Rtona	$@<MATCH>
Rtonh	$@<MATCH>
Rtont	$@<MATCH>
Rtooa	$@<MATCH>
Rtooh	$@<MATCH>
Rtoot	$@<MATCH>
Roda	$@<MATCH>
Rodh	$@<MATCH>
Rodt	$@<MATCH>
Roga	$@<MATCH>
Rogh	$@<MATCH>
Rogt	$@<MATCH>
Roia	$@<MATCH>
Roih	$@<MATCH>
Roit	$@<MATCH>
Rona	$@<MATCH>
Ronh	$@<MATCH>
Ront	$@<MATCH>
Rooa	$@<MATCH>
Rooh	$@<MATCH>
Root	$@<MATCH>
Rodda	$@<MATCH>
Roddh	$@<MATCH>
Roddt	$@<MATCH>
Rodga	$@<MATCH>
Rodgh	$@<MATCH>
Rodgt	$@<MATCH>
Rodia	$@<MATCH>
Rodih	$@<MATCH>
Rodit	$@<MATCH>
Rodna	$@<MATCH>
Rodnh	$@<MATCH>
Rodnt	$@<MATCH>
Rodoa	$@<MATCH>
Rodoh	$@<MATCH>
Rodot	$@<MATCH>
Rogda	$@<MATCH>
Rogdh	$@<MATCH>
Rogdt	$@<MATCH>
Rogga	$@<MATCH>
Roggh	$@<MATCH>
Roggt	$@<MATCH>
Rogia	$@<MATCH>
Rogih	$@<MATCH>
Rogit	$@<MATCH>
Rogna	$@<MATCH>
Rognh	$@<MATCH>
Rognt	$@<MATCH>
Rogoa	$@<MATCH>
Rogoh	$@<MATCH>
Rogot	$@<MATCH>
Roida	$@<MATCH>
Roidh	$@<MATCH>
Roidt	$@<MATCH>
Roiga	$@<MATCH>
Roigh	$@<MATCH>
Roigt	$@<MATCH>
Roiia	$@<MATCH>
Roiih	$@<MATCH>
Roiit	$@<MATCH>
Roina	$@<MATCH>
Roinh	$@<MATCH>
Roint	$@<MATCH>
Roioa	$@<MATCH>
Roioh	$@<MATCH>
Roiot	$@<MATCH>
Ronda	$@<MATCH>
Rondh	$@<MATCH>
Rondt	$@<MATCH>
Ronga	$@<MATCH>
Rongh	$@<MATCH>
Rongt	$@<MATCH>
Ronia	$@<MATCH>
Ronih	$@<MATCH>
Ronit	$@<MATCH>
Ronna	$@<MATCH>
Ronnh	$@<MATCH>
Ronnt	$@<MATCH>
Ronoa	$@<MATCH>
Ronoh	$@<MATCH>
Ronot	$@<MATCH>
Rooda	$@<MATCH>
Roodh	$@<MATCH>
Roodt	$@<MATCH>
Rooga	$@<MATCH>
Roogh	$@<MATCH>
Roogt	$@<MATCH>
Rooia	$@<MATCH>
Rooih	$@<MATCH>
Rooit	$@<MATCH>
Roona	$@<MATCH>
Roonh	$@<MATCH>
Roont	$@<MATCH>
Roooa	$@<MATCH>
Roooh	$@<MATCH>
Rooot	$@<MATCH>
Rpda	$@<MATCH>
Rpdh	$@<MATCH>
Rpdt	$@<MATCH>
Rpga	$@<MATCH>
Rpgh	$@<MATCH>
Rpgt	$@<MATCH>
Rpia	$@<MATCH>
Rpih	$@<MATCH>
Rpit	$@<MATCH>
Rpna	$@<MATCH>
Rpnh	$@<MATCH>
Rpnt	$@<MATCH>
Rpoa	$@<MATCH>
Rpoh	$@<MATCH>
Rpot	$@<MATCH>
Rpdda	$@<MATCH>
Rpddh	$@<MATCH>
Rpddt	$@<MATCH>
Rpdga	$@<MATCH>
Rpdgh	$@<MATCH>
Rpdgt	$@<MATCH>
Rpdia	$@<MATCH>
Rpdih	$@<MATCH>
Rpdit	$@<MATCH>
Rpdna	$@<MATCH>
Rpdnh	$@<MATCH>
Rpdnt	$@<MATCH>
Rpdoa	$@<MATCH>
Rpdoh	$@<MATCH>
Rpdot	$@<MATCH>
Rpgda	$@<MATCH>
Rpgdh	$@<MATCH>
Rpgdt	$@<MATCH>
Rpgga	$@<MATCH>
Rpggh	$@<MATCH>
Rpggt	$@<MATCH>
Rpgia	$@<MATCH>
Rpgih	$@<MATCH>
Rpgit	$@<MATCH>
Rpgna	$@<MATCH>
Rpgnh	$@<MATCH>
Rpgnt	$@<MATCH>
Rpgoa	$@<MATCH>
Rpgoh	$@<MATCH>
Rpgot	$@<MATCH>
Rpida	$@<MATCH>
Rpidh	$@<MATCH>
Rpidt	$@<MATCH>
Rpiga	$@<MATCH>
Rpigh	$@<MATCH>
Rpigt	$@<MATCH>
Rpiia	$@<MATCH>
Rpiih	$@<MATCH>
Rpiit	$@<MATCH>
Rpina	$@<MATCH>
Rpinh	$@<MATCH>
Rpint	$@<MATCH>
Rpioa	$@<MATCH>
Rpioh	$@<MATCH>
Rpiot	$@<MATCH>
Rpnda	$@<MATCH>
Rpndh	$@<MATCH>
Rpndt	$@<MATCH>
Rpnga	$@<MATCH>
Rpngh	$@<MATCH>
Rpngt	$@<MATCH>
Rpnia	$@<MATCH>
Rpnih	$@<MATCH>
Rpnit	$@<MATCH>
Rpnna	$@<MATCH>
Rpnnh	$@<MATCH>
Rpnnt	$@<MATCH>
Rpnoa	$@<MATCH>
Rpnoh	$@<MATCH>
Rpnot	$@<MATCH>
Rpoda	$@<MATCH>
Rpodh	$@<MATCH>
Rpodt	$@<MATCH>
Rpoga	$@<MATCH>
Rpogh	$@<MATCH>
Rpogt	$@<MATCH>
Rpoia	$@<MATCH>
Rpoih	$@<MATCH>
Rpoit	$@<MATCH>
Rpona	$@<MATCH>
Rponh	$@<MATCH>
Rpont	$@<MATCH>
Rpooa	$@<MATCH>
Rpooh	$@<MATCH>
Rpoot	$@<MATCH>
R$*	$@<NOMATCH>
